resources/grafana/rhacs-central-slo-dashboard.yaml

```diff
@@ -521,35 +536,73 @@ spec:
                   "Time": true,
                   "Time 1": true,
                   "Time 2": true,
+                  "Time 3": true,
                   "Value": false,
+                  "Value #C": true,
                   "__name__": true,
+                  "__name__ 1": true,
+                  "__name__ 2": true,
+                  "apiserver": true,
+                  "container": true,
+                  "endpoint": true,
+                  "instance": true,
+                  "job": true,
+                  "metrics_path": true,
                   "namespace": true,
+                  "namespace 1": true,
+                  "namespace 2": true,
+                  "node": true,
                   "observability": true,
+                  "pod": true,
+                  "prometheus": true,
+                  "prometheus_replica": true,
                   "rhacs_cluster_name": true,
-                  "rhacs_environment": true
+                  "rhacs_environment": true,
+                  "rhacs_instance_type": true,
+                  "rhacs_version": false,
+                  "service": true
                 },
                 "indexByName": {
-                  "Time 1": 10,
-                  "Time 2": 11,
+                  "Time 1": 9,
+                  "Time 2": 10,
+                  "Time 3": 13,
                   "Value #A": 1,
                   "Value #B": 2,
-                  "__name__": 5,
-                  "namespace": 7,
-                  "observability": 8,
+                  "Value #C": 28,
```

Comment:
Are these generated changes? If not, what is the purpose behind them? Talking about lines https://github.com/stackrox/rhacs-observability-resources/pull/118/files#diff-cec7c1223ca9d5671a37727817b570e4bc72df0cd59f33280bb7e5656d354020R539-R605

Reply:
The entire json is generated, however it's based on the changes that I made via the Grafana UI. The process is basically like this:
* Load the dashboard in Grafana.
* Make the changes you want, like updating/adding/removing widgets.
* Export the entire dashboard as json and paste it into the PR.

Specifically, what you are looking at here is the diff generated by adding another query to the table view. You can select all labels of the result as columns in the table. In this case I hid most of them and only select org name and org id.